src/Components/LocationForm/LocationForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
```

Comment:
I'm not an expert in forms using React, but it seems to me that will be easier to keep track of the app workflow/state if you use the concept of Controlled Components, where the main app manage the form state, then you don't need to keep each state in its own component, which makes the code more difficult to reason about.

This is not a blocking comment, as I will approve the PR still, but maybe something to study!

Reply:
I believe I thought the wrong way, trying to create an independent form component with its own state to control inputs. But for the sake of clarity, it's better to lift state up to App.js. 
We can fix this and add other improvements after the merge.